doc/doxygen/headers/glossary.h

```diff
@@ -1013,49 +1013,71 @@
+ * faces 1 and 3 have normals pointing outward. In 3d (for
+ * ReferenceCells::Hexahedron), faces 0, 2, and 4 have normals which point into
+ * the cell whereas the normals of faces 1, 3, and 5 point outward. This
+ * information may be queried via ReferenceCell::face_normal_vector().
+ *
+ * For most unstructured hexahedral meshes and essentially all simplex meshes,
+ * there is no way to order the vertices (and thereby order the faces) such that
+ * every cell has both a positive Jacobian and consistent normal vector
+ * orientations. This is due to the fact that the face convention for one cell
+ * already implies something for the neighbor, since they share a common face
+ * and fixing it for the first cell also fixes the normal vectors of the
+ * opposite faces of both cells. It is easy to construct cases of loops of cells
+ * for which this leads to cases where we cannot find orientations for all faces
+ * that are consistent with this convention.
+ *
+ * For this reason, the above convention is what we call the <em>standard
+ * orientation</em> deal.II allows faces to have either the default direction or
+ * its opposite. TriaAccessor::face_orientation() returns `true` for a given
+ * face if that face is in the default direction and `false` otherwise. As faces
+ * may also be rotated it is generally more useful to use the @ref
+ * GlossCombinedOrientation "combined orientation" to cover all possible
+ * rotations and orientations.
  * </dd>
  *
+ * <dt class="glossary">@anchor GlossCombinedOrientation <b>Combined
+ * orientation</b></dt>
+ * <dd>
+ * A Triangulation contains cells as well as lower dimensional objects such as
+ * faces (which are either lines in 2d or quadrilaterals or triangles in 3d). In
+ * general, the vertices of each cell are numbered in a way which results in a
+ * mapping with a positive Jacobian. A consequence of this choice is that the
+ * vertices which define a face may be, from the perspective of an arbitrary
+ * cell, in a different order than the order given to that face by the
+ * neighboring cell. To resolve this inconsistency deal.II stores, for each face
+ * and (in 3d) line, a value which may be used to permute the vertices on both
+ * faces into a matching configuration. This encoding contains both the number
+ * of times a face should be rotated (relative to its neighbor) as well as the
+ * orientation value discussed in @ref GlossFaceOrientation "the definition of
+ * face orientations".
+ *
+ * This value is called the <em>combined_orientation</em>. In some
+ * circumstances, to disambiguate between faces and lines, it may also be called
+ * either the `combined_face_orientation` or the `combined_line_orientation`.
+ * These orientations are represented by types::geometric_orientation, which
+ * encodes both the @ref GlossFaceOrientation "face orientation" as well as the
+ * number of times it a face should be rotated. This encoding is an internal
```

Comment:
```suggestion
 * number of times a face should be rotated so that the face's representation in the cell's coordinate system matches the face's own coordinate system. This encoding is an internal
```